diff -uNr grandr_applet-0.4.1.orig/GrandrApplet.server.in grandr_applet-0.4.1/GrandrApplet.server.in
--- grandr_applet-0.4.1.orig/GrandrApplet.server.in	2007-12-13 15:14:33.000000000 +0000
+++ grandr_applet-0.4.1/GrandrApplet.server.in	2009-03-01 14:58:31.000000000 +0000
@@ -2,7 +2,7 @@
 
 <oaf_server iid="OAFIID:GrandrApplet_factory" 
 	    type="exe"
-            location="@prefix@/libexec/grandr">
+            location="@libexecdir@/grandr">
 
         <oaf_attribute name="repo_ids" type="stringv">
                 <item value="IDL:Bonobo/GenericFactory:1.0"/>
